flycheck.el

```diff
@@ -8060,12 +8060,14 @@ for more information about the custom directories."
   :safe #'flycheck-string-list-p
   :package-version '(flycheck . "29"))
 
-(defun flycheck-eslint-config-exists-p ()
-  "Whether there is an eslint config for the current buffer."
-  (let* ((executable (flycheck-find-checker-executable 'javascript-eslint))
-         (exitcode (and executable (call-process executable nil nil nil
-                                                 "--print-config" "."))))
-    (eq exitcode 0)))
+(defun flycheck-eslint-config-error ()
+  "String of eslint config error for the current buffer."
+  (let ((executable (flycheck-find-checker-executable 'javascript-eslint))
+        (stderr-file (flycheck-temp-file-system "eslint-config-errors")))
+    (call-process executable nil `(nil ,stderr-file) nil "--print-config" ".")
```

Comment:
Don't you still need to check for the executable? Won't this raise an error if it can't be found?

Reply:
Good point! i broke that!